app/src/androidTest/java/com/android/sample/utils/RecipeListTest.kt

```diff
@@ -123,7 +123,7 @@ class RecipeListTest {
           list = recipesList,
           onRecipeSelected = {},
           topCornerButton = { recipe ->
-            TopCornerLikeButton(recipe = recipe, userViewModel = userViewModel)
+            TopCornerLikeButton(recipe = recipe, userViewModel = userViewModel, true)
```

Comment:
This is a continuation of my earlier comment on `app/src/main/java/com/android/sample/ui/utils/RecipeList.kt.` To improve readability and maintainability, you could simplify this method by splitting it into two separate methods. Here's a suggestion:
```suggestion
            TopCornerDeleteButton(recipe = recipe, userViewModel = userViewModel)
```

Reply:
Done